romana-install/roles/romana/kubernetes-postinstall/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Create bin directory
+  file: path="/home/ubuntu/bin/" mode=0775 state=directory
+
+- name: Install romana tool
+  template: src="../cmd/romana.k8s" dest="/home/ubuntu/bin/romana" mode=0775
+
+- name: Install romana post-install script
+  template: src="romana-post-install.sh" dest="/var/tmp/romana-post-install.sh" mode=0755
+        
+- name: Execute romana post-install script
+  shell: /var/tmp/romana-post-install.sh
+
+- name: Create demo directory
+  file: path="/home/ubuntu/demo/" mode=0775 state=directory
+
+- name: Install additional tools
+  become: true
+  become_user: root
+  apt: pkg="{{ item }}"
+  with_items:
+    - tmux
```

Comment:
These items are only needed for our demo, are they not? We should probably mention that in the comment.